### PR TITLE
Changed copy function to use ncp isntead of wrench

### PR DESCRIPTION
**Pull request by fewstera**
This PR should allow our `grunt deploy:live` task to work again :+1:

### CODE REVIEW

File: lib/deploy.js [hunk 1]
```diff
@@ -28,11 +29,13 @@ module.exports = function () {
         deploy: function (options, done) {
             this.copyOutputToTemporaryDirectory(options);
             this.findAndReplaceEnvironments(options);
-            this.copyToDeploymentZone(options);
-            this.cleanUpTemporaryFiles();
 
-            console.log('Content deployed to following location: ' + options['server'] + options['path']);
-            done();
+            this.copyToDeploymentZone(options, function () {
+                deploy.cleanUpTemporaryFiles();
```
Comment (ChrisBAshton) [inline lib/deploy.js]: If I'm being picky, I'd rename `deploy` to `Deploy` on line 11 and then reference `Deploy` here on line 34. Makes it clearer what the scope is (i.e. we're not referring to the `deploy` function on line 29)

Comment (ChrisBAshton) [inline lib/deploy.js]: Actually, scrap that - I'd remove the call to `cleanUpTemporaryFiles` altogether, and just put `Deploy.deleteRecursive(tmpDir);` here

Comment (ChrisBAshton) [inline lib/deploy.js]: And name the anonymous function callback in line 33, make it `options, function tidyUp () {`

Comment (fewstera) [inline lib/deploy.js]: I'd be opposed to using Deploy, as a capital letter usually references a prototype not an object.

Comment (fewstera) [inline lib/deploy.js]: Renaming `deploy` to `task` could work though? `task.deploy` `task.deleteRecursive`

**Comment by ChrisBAshton** [comment]
Fantastic mate, tried it out with a test project and it works a treat :+1:

File: lib/deploy.js (hunk 1 above)
Comment (ChrisBAshton) [inline lib/deploy.js]: Sure, sounds good

**Pull request merged**